h/templates/activity/search.html.jinja2

```diff
@@ -76,6 +76,11 @@
            target="_blank">{{ pretty_link(bucket.uri) }}</a>
     </div>
   {% endif %}
+  <div class="u-stretch">
+  </div>
+  <a data-ref="collapseView" class="search-bucket-stats__collapse-view">
```

Comment:
I would not use anchor elements without hrefs for this. Use `<button>`s when you need a non document linking click target.